test/models/like_test.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
```

Comment:
- [**OPTIONAL**] **RECOMMENDED** `In README.md file:`
 I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or another method change this `[MIT](./LICENSE)` to  `[MIT](./MIT.md)` then the created new file in your directory name it `MIT.md` and get its content **[here](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

Reply:
Of course! Thank you very much for your advice. I will take them all into account!